javascript/src/main/App.js

```diff
@@ -48,6 +49,7 @@ function App() {
           <AuthorizedRoute path="/member/channels" exact component={ChannelList} authorizedRoles={["admin","member"]} />
           <AuthorizedRoute path="/member/search" exact component={Search} authorizedRoles={["admin","member"]} />
           <AuthorizedRoute path="/member/searchhistory" exact component={SearchHistory} authorizedRoles={["admin","member"]} />
+          <AuthorizedRoute path="/member/searchhistory/alluser" exact component={Alluser} authorizedRoles={["admin","member"]} />
```

Comment:
The page to view the search history for all members should only be visible to admins.

Reply:
Fix this error.